src/client/components/Description Career/descriptioncareer.css

```diff
@@ -0,0 +1,39 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400&display=swap');
+
+.descriptioncareer {
+  padding: 10px;
+  padding-left: 20px;
+  background-color: #e9f4f4;
+  margin: 10px;
+}
+
+.innerDiv {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  justify-content: space-between;
+}
+
+.descriptioncareer .title {
+  color: #db643d;
+  font-family: 'Roboto Mono', monospace;
+  font-style: normal;
+  font-size: 28px;
```

Comment:
One more thing, if you change the font-size into 25px, it looks more identical with the actual design I think :)

Reply:
thank you @TITHI1244  will do that 👍